src/formats/html.js

```diff
@@ -41,10 +41,10 @@ module.exports = function ( reports ) {
 	twing
 		.render( reportTemplate, context )
 		.then( ( output ) => {
-			console.log( `Generated template for ${ reportName }.` );
+			console.log( `Generated template for ${ reportName }.` ); // eslint-disable-line no-console
```

Comment:
These `eslint-disable-line` exceptions can be removed, since we allow console everywhere now https://github.com/ryelle/css-audit/commit/c706bb951d0212631038e327497220216beabc68